frontend/src/data/config/MetadataMap.ts

```diff
@@ -1,18 +1,10 @@
-import { DatasetMetadata, MetadataMap } from "../utils/DatasetTypes";
+import { DatasetMetadata, DataSourceMetadata } from "../utils/DatasetTypes";
 
-const fakeMetadata: DatasetMetadata[] = [
+const datasetMetadataList: DatasetMetadata[] = [
```

Comment:
To improve the distinction between what's fake vs permanent, could we move this to another file and name it FakeDatasetMetadata?